Programming/Installing-WPILib.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Installing WPILib
+parent: Tools
+grand_parent: Programming
+has_children: false
+layout: home
+---
+
+The WPILib is the core FRC library that the programming team uses to create code for the robots. By installing the WPILib you get:
+- VSCode, which is an integrated development environment (IDE) that works best with the WPILib
+- Java JDK
+- Gradle
+- Tools (SmartDashboard, Shuffleboard, etc.)
+
+Follow the instructions in the [WPILib Docs](https://docs.wpilib.org/en/stable/docs/zero-to-robot/step-2/wpilib-setup.html) to install the WPILib and VS Code (we use the recommended settings).
```

Comment:
@linglejack06 just saw your email - I added this page to our docs. I would just follow the instructions in this URL directly. We use VS Code since it's fully integrated with WPILib